samples/drivers/video/capture/Kconfig

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2024 Espressif Systems (Shanghai) Co., Ltd.
+# SPDX-License-Identifier: Apache-2.0
+
+mainmenu "Video capture sample application"
+
+menu "Video capture configuration"
+
+config VIDEO_FORMAT_HEIGHT
```

Comment:
isn't it useful to limit maximum values for height and width?

Reply:
I'm not sure. If we add a limit, it should be based on the sensor capabilities. If a given resolution is not supported the sample already returns an error with  the proper message. What do you think?

Reply:
Overly large buffer sizes would fail during allocation (or rather block forever):
https://github.com/zephyrproject-rtos/zephyr/pull/77879/files#diff-fd410d45441d4e562a8cdb14320e58f2168f446e0fad0b4bb54cedd7cef29b6fR193

There is a separate `Kconfig` that will limit the total amount of memory the whole video drivers can use:
https://docs.zephyrproject.org/latest/kconfig.html#CONFIG_VIDEO_BUFFER_POOL_SZ_MAX